src/gui_tool/entry.py

```diff
@@ -45,8 +49,29 @@ def split_file(file_loc, mdi:MetaDataItem):
         try:
             context = GUIContext(file_loc)
             gui = SPGUIManager(context)
-            gui.start()
-            return mdi
+            rs = gui.start()
+            ret = [mdi]
+
+            split_vid = False
```

Comment:
Meant to put this in a separate branch to pull into this one.

Reply:
@tonymeng1 do you see any problems with these suggested changes, if so I'll revert this commit.